spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/AbstractMessageBusBinderPlugin.java

```diff
@@ -157,6 +218,28 @@ else if (entry.getKey().startsWith(producerKeyPrefix)) {
 		return new Properties[] { consumerProperties, producerProperties };
 	}
 
+	protected final Map<String, Object> extractHistoryProperties(Module module) {
+		Map<String, Object> properties = new LinkedHashMap<String, Object>();
+		if (module.getProperties() != null) {
+			for (Map.Entry<Object, Object> entry : module.getProperties().entrySet()) {
+				if (entry.getKey() instanceof String) {
+					String key = (String) entry.getKey();
+					if (key.startsWith(ModulePlaceholders.XD_CONTAINER_KEY_PREFIX)) {
+						key = key.substring(ModulePlaceholders.XD_CONTAINER_KEY_PREFIX.length());
+						if (key.equals("id")) {
```

Comment:
It seemed a little verbose to have `xd.container.` on every tag; most of the other container properties (host, ip etc) are self-describing but `id` didn't convey anything to me. I also didn't want it to be confused with the `id` header which is also a UUID.